addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -221,7 +227,7 @@ class Composer extends Component {
 
         const context = {
             // default_parent_id: this.id,
-            default_body: this._textInputRef.comp.getHtmlContent(),
+            default_body: mailUtils.getHtmlContent(this._textInputRef.comp.getContent()),
             default_attachment_ids: attachmentIds,
             // default_partner_ids: partnerIds,
             default_is_log: this.props.isLog,
```

Comment:
lines 272 & 345:
instead of `this._textInputRef.comp.isEmpty`, use `this.storeProps.composer.textInputContent` instead

Reply:
Done